03-monitoring-an-app/002-how-to-create-alarms.md

```diff
@@ -0,0 +1,99 @@
+---
+category: cloud-platform
+expires: 2019-07-01
+---
+# Creating your own custom alerts
+
+## Overview
+Alertmanager allows you define your own alert conditions based on [Prometheus expression language](https://prometheus.io/docs/prometheus/latest/querying/basics) expressions. 
+
+The aim of this document is to provide you with the necessary information to create and send application specific alerts to a Slack channel of your choosing.
+
+## Prerequisites
+This guide assumes the following:
+
+* You have [created a namespace for your application]({{ "/01-getting-started/002-env-create" | relative_url }})
+* Deployed at least one pod in your namespace 
```

Comment:
Why is the `Pod` required?

Reply:
Agreed. I'll remove it.

Reply:
Removed in 
[15e81c5](/ministryofjustice/cloud-platform-user-docs/pull/114/commits/15e81c508f1a6b321666886927136e91d82f6925)